api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -112,6 +113,16 @@ public void setEnableSaramaLogging(boolean enableSaramaLogging) {
         this.enableSaramaLogging = enableSaramaLogging;
     }
 
+    @Description("Enable/disable offset show all option.")
```

Comment:
I would be more explicit in the description. Maybe we can reuse what's in the official Kafka Exporter "documentation"? 

> Whether show the offset/lag for all consumer group, otherwise, only show connected consumer groups